src/plugins/vis_type_table/public/components/table_visualization.scss

```diff
@@ -12,6 +12,13 @@
   overflow: auto;
 
   @include euiScrollBar;
+
+  // Sticky footer doesn't correct work with inline-flex in Firefox.
+  // As footer the last element I don't see any reason to use inline-flex for this element.
+  // Display: flex fixes jumping on hover in Firefox.
```

Comment:
Please add a comment with a link to this related EUI issue - https://github.com/elastic/eui/issues/4729

Once addressed, we can remove this local fix

Reply:
@ryankeairns @chandlerprall FYI setting 

```
.euiDataGridHeader
{
 display: flex;
}
```

also resolves the header position problem in Firefox, just tried it out, works

Reply:
@ryankeairns, added comment.